Cargo.toml

```diff
@@ -3,36 +3,37 @@ name = "nom-derive"
 description = "Custom derive nom parsers from struct"
 version = "0.10.0"
 license = "MIT/Apache-2.0"
-keywords = ["parser","nom"]
+keywords = ["parser", "nom"]
 categories = ["parsing"]
 authors = ["Pierre Chifflier <chifflier@wzdftpd.net>"]
 homepage = "https://github.com/rust-bakery/nom-derive"
 repository = "https://github.com/rust-bakery/nom-derive.git"
 readme = "README.md"
-edition = "2018"
+edition = "2024"
```

Comment:
I have to make more tests, I am not sure moving to edition 2024 is required nor useful. My concern is that I want this crate to stay compatible with older code.
It may be best to keep an old edition but add an MSRV (`rust_version = "1.65"`)